### DISCUSSION TITLE
users export dont work ,error

### DISCUSSION THREAD

**Discussion by rioshaz** [YetiForceCompany/YetiForceCRM#14491] [General]
i am unable to export user list...
check on gistable too..
![image](https://user-images.githubusercontent.com/54880467/113412232-606ceb00-93d5-11eb-92e2-52fc5408fa44.png)

**Comment by mariuszkrzaczkowski** [comment]
check on gitdev maybe the bug has already been fixed

**Reply by rioshaz** [reply]
okay but how do i fix it on my system ?  can u guide?

**Reply by rioshaz** [reply]
> check on gitdev maybe the bug has already been fixed

when can we expect 6.2

**Reply by rioshaz** [reply]
> check on gitdev maybe the bug has already been fixed

this is no answer to someone who is using 6.1...

**Comment by rioshaz** [comment]
so we have to wait till 6.2.. or can u pls share the fix ?

**Comment by vovpff** [comment]
As temporary solution you can export data directy from database

**Reply by rioshaz** [reply]
i wanna import

**Reply by vovpff** [reply]
Ok, but you wrote what export broken? I'm right?

**Reply by rioshaz** [reply]
yes thats true because only after export i can have the excel template to import..

**Reply by rioshaz** [reply]
so as informed by @mariuszkrzaczkowski  i think i can download export template from gitdev and then create my excel and import here may be ti works,.. but still who is using yf n needs to exports 100's of user its very diccult

**Reply by mariuszkrzaczkowski** [reply]
if someone has hundreds of users, they usually have support and are not struggling with such problems

**Reply by rioshaz** [reply]
> if someone has hundreds of users, they usually have support and are not struggling with such problems

ooh so yf only gives fixes bugs to high-end clients ... we testers have no real value.. good to know sir .... .

**Reply by mariuszkrzaczkowski** [reply]
No, the support is for those who don't want the headache of solving problems/bugs.
We only spend time on GitHub when we have it, so it may take some time before we respond.
When someone expects a response/fix from us after a few hours, it happens only via SLA tickets, not GitHub issues.
We are discouraged from writing back when someone throws us with mud or expresses a negative attitude.

**Reply by mariuszkrzaczkowski** [reply]
see bug reporting requirements  https://yetiforce.com/en/knowledge-base/documentation/implementer-documentation/item/how-to-report-bugs

**Reply by rioshaz** [reply]
> No, the support is for those who don't want the headache of solving problems/bugs.
> We only spend time on GitHub when we have it, so it may take some time before we respond.
> When someone expects a response/fix from us after a few hours, it happens only via SLA tickets, not GitHub issues.
> We are discouraged from writing back when someone throws us with mud or expresses a negative attitude.

no one's throwing mud t you or any negative comment.. we appreciate your hard work n support ..
what my point is saying the issue has been fixed in 6.2 isnt really a fix...

Further, yes to report bug there is specific format but if u have understood the problem i dont see a point in asking to  forcefully get that format from a user...

no matter ehat you help n support is always appreciated.. thank u buddy

**Comment by vovpff** [comment]
There is track of error:

```
app/Purifier.php(468) #0 app/Request.php(168): App\Purifier::purifyByType('', 2, false) #1 modules/Vtiger/views/Export.php(35): App\Request->getByType('viewname', 2) #2 include/main/WebUI.php(189): Vtiger_Export_View->process(Object(App\Request)) #3 index.php(35): Vtiger_WebUI->process(Object(App\Request)) #4 public_html/index.php(11): require('...') #5 {main}
```

**Reply by rskrzypczak** [reply]
It's been fixed long ago, see: https://github.com/YetiForceCompany/YetiForceCRM/issues/13927
